modules/developer-cli-odo-installing-odo-on-macos.adoc

```diff
@@ -7,24 +7,25 @@
 
 = Installing {odo-title} on macOS
 
-The `{odo-title}` CLI for macOS is available to download as a binary and as a tarball.
+The `{odo-title}` CLI for macOS is available to download as a binary and as a tar file.
 
-[cols="2,1,1",options="header"]
+[cols="1,1,1",options="header"]
 |===
-|Operating System|Binary|Tarball
-|macOS|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64[odo-darwin-amd64]|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64.tar.gz[odo-darwin-amd64.tar.gz]
+|Operating system|Binary|Tar file
+|macOS on AMD|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64[odo-darwin-amd64]|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64.tar.gz[odo-darwin-amd64.tar.gz]
+|macOS on ARM|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-arm64[odo-darwin-arm64]|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-arm64.tar.gz[odo-darwin-arm64.tar.gz]
 |===
 
 .Procedure
 
-. Navigate to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file:
+. Browse to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file:
```

Comment:
See IBM SG "Referring to web addresses" information.

Reply:
Ok, I will change to "Go to the content gateway and download...." seeing as that is what the IBM SG says

Reply:
Done